### PR TITLE
Event targets pop-up window

### PR DESCRIPTION
**Pull request by sonishc**
bootstrap color

- [ ] @afurm 
- [ ] @rubycop 
- [ ] @SetterVet 
- [ ] @VasylYaremkiv

### CODE REVIEW

**Comment by SetterVet** [comment]
Also, add this Vue component to Edit Event button

**Comment by SetterVet** [comment]
Add tests

**Comment by rubycop** [comment]
Make styling for pop-up window

File: app/javascript/components/users/UserInfo.vue [hunk 1]
```diff
@@ -1,34 +1,96 @@
...
-    </div>
-    <div class="col-6">
-      <div class="card">
-        <div class="card-body">
-          <h5 class="card-title">{{ user.project.name }}</h5>
-          <p class="card-text">{{ user.project.desc }}</p>
+  <div>
+    <div class="row">
+      <div class="col-6">
+        <h3>{{ user.first_name + " " +user.last_name }}</h3>
```
Comment (VasylYaremkiv) [inline app/javascript/components/users/UserInfo.vue]: missed space

**Pull request merged**